README.md

```diff
@@ -14,102 +14,110 @@ library can help with that!
 To get started, you will need a [GitHub Personal Auth
 Token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
 to make requests to the GitHub API. This should be set in your environment or
-shell ``rc`` file with the name ``GITHUB_API_TOKEN``:
+shell `rc` file with the name `GITHUB_API_TOKEN`:
 
+```shell
     $ export GITHUB_API_TOKEN=XYZ
 
     $ echo "export GITHUB_API_TOKEN=XYZ" >> ~/.bashrc
+```
 
 Additionally, to perform the labor hours estimation, you will need to install
-``cloc`` into your environment. This is typically done with a [Package
+`cloc` into your environment. This is typically done with a [Package
 Manager](https://github.com/AlDanial/cloc#install-via-package-manager) such as
-``npm`` or ``homebrew``.
+`npm` or `homebrew`.
 
-Then to generate a ``code.json`` file for your agency, you will need a
-``config.json`` file to coordinate the platforms you will connect to and scrape
+Then to generate a `code.json` file for your agency, you will need a
+`config.json` file to coordinate the platforms you will connect to and scrape
 data from. An example config file can be found in [demo.json](/demo.json). Once
 you have your config file, you are ready to install and run the scraper!
 
-    # Install Scraper
+```shell
+    # Install Scraper from a local copy of this repository
     $ pip install -e .
+    # OR
+    # Install Scraper from PyPI
+    $ pip install llnl-scraper
 
     # Run Scraper with your config file ``config.json``
     $ scraper --config config.json
+```
 
-A full example of the resulting ``code.json`` file can be [found
+A full example of the resulting `code.json` file can be [found
 here](https://gist.github.com/IanLee1521/b7d7c0c2d8c24b10dd04edd5e8cab6c4).
 
 ## Config File Options
 
 The configuration file is a json file that specifies what repository platforms
-to pull projects from  as well as some settings that can be used to override
+to pull projects from as well as some settings that can be used to override
 incomplete or inaccurate data returned via the scraping.
 
 The basic structure is:
 
-```json
+```jsonc
```

Comment:
Nice! Is `.jsonc` an actual file type / flavor that parsers know how to handle? I know some are very picky...

Reply:
It seems to be relatively common. At worst, I think most markdown parsers tend to quietly overlook any unknown syntax specifications and just display the mono-spaced code block without any special highlighting. I think that's still an improvement over the angry red error highlights from the invalid json.

Reply:
agreed!